recipes/ibis-framework/meta.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: ibis-framework
+  version: "SETBYINSTALL"
+
+source:
+  git_url: https://github.com/cloudera/ibis
+  git_rev: v0.7.1
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest
+    - numpy >=1.7.0
+    - pandas >=0.12.0
+    - impyla >=0.13.6
+    - hdfs >=2.0.0
+    - sqlalchemy >=1.0.0
+    - six
+    - mock
```

Comment:
Our build matrix will try to build Python 3.5 and this may fail as mock is part of the standard library now. You can add a `#  [not py35]` after it to avoid problems.